include/rcutils/macros.h

```diff
@@ -57,7 +57,12 @@ extern "C"
   #endif
 #else
 // Some other non-Windows, non-cygwin, non-apple OS
-  #define RCUTILS_THREAD_LOCAL _Thread_local
+  #if  __STDC_VERSION__ >= 201112L
+// C11 enabled
```

Comment:
How so? Doesn't this follow the same style as https://github.com/ros2/rcutils/blob/master/include/rcutils/macros.h#L32 https://github.com/ros2/rcutils/blob/master/include/rcutils/macros.h#L35 et al?